python/fledge/common/logger.py

```diff
@@ -76,19 +82,27 @@ def _get_process_name():
 
     logger = logging.getLogger(logger_name)
 
+    # if no destination is set, use the fledge default
+    if destination == None:
+        destination = default_destination
+
     if destination == SYSLOG:
         handler = SysLogHandler(address='/dev/log')
     elif destination == CONSOLE:
-        handler = logging.StreamHandler(sys.stdout)
+        handler = logging.StreamHandler(sys.stderr)
```

Comment:
probably we can remove this elif, default StreamHandler is stderr

Reply:
done